doc/api/globals.md

```diff
@@ -498,6 +498,18 @@ changes:
 
 A browser-compatible implementation of [`DecompressionStream`][].
 
+## `ErrorEvent`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+<!-- type=global -->
+
+The `ErrorEvent` class. See [`ErrorEvent`][] for more details.
+
+A browser-compatible implementation of [`ErrorEvent`][].
```

Comment:
nit: it would probably make more sense to link it as a type. Also, do we want to mark it as experimental?

```suggestion
A browser-compatible implementation of {ErrorEvent}.
```

Reply:
might be inconsistent with https://github.com/nodejs/node/pull/53355/files#diff-27d988075d57cd32dd527269eebc991f4567f1bf7a751a39ab72e2ae70050031R335